doc/lua_api.md

```diff
@@ -7820,7 +7822,11 @@ Global tables
       Note: changes to initial properties will only affect entities spawned afterwards,
       as they are only read when spawning.
 * `core.object_refs`
-    * Map of object references, indexed by active object id
+    * Map of active object references, indexed by active object id
+    * Obsolete: Use `core.objects_by_guid` instead.
```

Comment:
maybe make it clearer that use of this table by mods is deprecated (obsoleted?). It seems unlikely that we will remove the table itself.